tests/frontend/unit/ExportModal.spec.js

```diff
@@ -0,0 +1,432 @@
+    const matchingTabContent = wrapper.find(`#${id}`)
+
+    expect(getComputedStyle(matchingTabContent.element).display).toBe('block')
+  })
+
+    it('triggers a "submit" event when the submit button is clicked', async () => {
+      wrapper.vm.submit = jest.fn()
+      wrapper.vm.$refs.exportModal = {
+        toggle: jest.fn()
+      }
+
+      wrapper.vm.handleSubmit()
+
+      const event = wrapper.emitted('submit')
+
+      expect(wrapper.emitted()).toHaveProperty('submit')
+      expect(event).toHaveLength(1)
+    })
+
+  describe('ExportModal: pdf export', () => {
+    it('displays "anonymized" checkbox in the pdf tab if pdf.anonymize or pdf.obscure option is set to true', async () => {
+      const tabButtons = wrapper.findAll('button')
+      const index = 1
+      await tabButtons.at(index).trigger('click')
+      const checkboxStub = wrapper.find('[datacy="exportModal:pdfAnonymous"]')
+
+      expect(checkboxStub.exists()).toBe(true)
+    })
+
+    it('does not display "anonymized" checkbox in the pdf tab if pdf.anonymize and pdf.obscure options are set to false', async () => {
+
+      await wrapper.setProps({
+        options: {
+          ...props.options,
+          pdf: {
+            ...props.options.pdf,
+            anonymize: false,
+            obscure: false
+          }
+        }
+      })
+
+      const checkboxStub = wrapper.find('[datacy="exportModal:pdfAnonymous"]')
+      expect(checkboxStub.exists()).toBe(false)
+    })
+
+    it('displays "new page per statement" checkbox in the pdf tab in the original statements view if "newPagePerStn" option is set to true', async () => {
+      await wrapper.setProps({
+        view: 'original_statements'
+      })
+
+      const checkboxStub = wrapper.find('[datacy="exportModal:newPagePerStn"]')
+      expect(checkboxStub.exists()).toBe(true)
+    })
+
+    it('does not display "new page per statement" checkbox in the pdf tab if "newPagePerStn" option is set to true, but view is "assessment_table"', async () => {
+      await wrapper.setProps({
+        view: 'assessment_table'
+      })
+
+      const checkboxStub = wrapper.find('[datacy="exportModal:newPagePerStn"]')
+      expect(checkboxStub.exists()).toBe(false)
+    })
+
+    it('does not display "new page per statement" checkbox in the pdf tab in the original statements view if "newPagePerStn" option is set to false', async () => {
+      await wrapper.setProps({
+        view: 'original_statements',
+        options: {
+          ...props.options,
+          pdf: {
+            ...props.options.pdf,
+            newPagePerStn: false
+          }
+        }
+      })
+
+      const checkboxStub = wrapper.find('[datacy="exportModal:newPagePerStn"]')
+      expect(checkboxStub.exists()).toBe(false)
+    })
+
+    it('displays radio buttons for selecting a format in the pdf tab if pdf.templates option is true', () => {
+      const radioButtons = wrapper.findAll('[datacy^="exportModal:pdfTemplate"]')
```

Comment:
```suggestion
      const radioButtons = wrapper.findAll('[datacy="exportModal:pdfTemplate"]')
```
I think this is a typo but since this can be found on other occurrences too, maybe this is a syntax I do not know?

Reply:
It's not a typo, it looks for all elements that have a `datacy` attribute that has a value that starts with the given string